src/components/report/Summary.js

```diff
@@ -8,6 +8,9 @@ export default {
 		};
 	},
 	computed: {
+		hasLocalization() {
+			return this.font.languageSystems.length > 0;
```

Comment:
Does it make sense that this is looking at `languageSystems`, instead of `languages`? Because ultimately it's rendering the `languages`, so you might as well check that?

Reply:
We have two concepts now: **languages** that can be written based on the characters in the font, and **localization** of characters inside the font. The latter is a mechanism in the font where a certain character can look different to match local conventions.  See more [here](https://www.fontsmith.com/blog/2016/10/12/cyrillic-script-variations-and-the-importance-of-localisation).

I renamed `languages` to `localizations` to make this distinction more clear. I think it's better now, thanks!

Reply:
Ah nice, thanks for the explanation!